docs/cicd.md

```diff
@@ -14,40 +14,40 @@
+  * Per default a development delivery is made.
+  * For more information on how to use it, execute it without parameters to see usage information.
 
 ## Workflows
 
 Several workflows exist for different purposes:
-* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat, Malte and Aschaffenburg webapps on the main branch.
-* **scheduled_native_development_delivery**: Delivers Integreat, Malte and Aschaffenburg native builds to development (Testflight and Beta channel)
-twice a month.
-* **triggered_native_development_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers Integreat, Malte and Aschaffenburg builds for native to development.
-* **scheduled_production_delivery**: Delivers Integreat, Malte and Aschaffenburg native builds to production twice a month.
-* **triggered_production_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers Integreat, Malte and Aschaffenburg builds for native and web to production.
+* **commit**: Executed for all commits of PRs to ensure good code quality and working code. Delivers web development builds to https.//webnext.\<domain>.
+* **scheduled_native_promotion**: Promotes the latest native development builds to production. Executed every Thursday morning.
+* **scheduled_delivery**: Delivers native builds to development and web builds to production. Executed every Thursday morning.
+* **triggered_native_development_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers native builds to development.
+* **triggered_production_delivery**: [Manually triggerable](#triggering-a-delivery-using-the-ci) workflow which delivers web and native builds to production.
 
 See the table below for a more detailed overview:
 
 |Workflow|Checks|E2E tests|native delivery|web delivery|Version bump|Move release notes|
 |---|---|---|---|---|---|---|
-|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|Only on main|:x:|:x:|
-|scheduled_native_development_delivery|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|
-|triggered_native_development_delivery|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|
-|scheduled_production_delivery|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
-|triggered_production_delivery|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|development (main only)|:x:|:x:|
+|scheduled_native_promotion|:x:|:x:|promotion|:x:|:x:|:x:|
+|scheduled_delivery|:heavy_check_mark:|:heavy_check_mark:|development|production|:heavy_check_mark:|:heavy_check_mark:|
+|triggered_native_development_delivery|:heavy_check_mark:|:heavy_check_mark:|development|:x:|:heavy_check_mark:|:x:|
```

Comment:
why does the `triggered_native_development_delivery` not move the release notes, but the `scheduled_delivery` does?

Reply:
Since scheduled_delivery does a release on all platforms (including web). If we would move the release notes in `triggered_native_development_delivery`, we would never have release notes of web in github releases and in mattermost afterwards.